plugins/outputs/influxdb_v2/README.md

```diff
@@ -30,6 +30,15 @@ The InfluxDB output plugin writes metrics to the [InfluxDB v2.x] HTTP service.
   ## If true, the bucket tag will not be added to the metric.
   # exclude_bucket_tag = false
 
+  ## If true, no create bucket requests will be sent. Set to true when using
+  ## Telegraf with a user without permissions to create buckets or when the
+  ## the bucket already exists
+  # skip_bucket_creation = false
```

Comment:
To make this backward compatible, we need to default to the plugin's previous behavior and let users opt in to get the new behavior. Also, using a negative setting (`skip_action = false`) is not as clear as a positive setting (`act = true`) for users writing their toml config and for devs reading the code.

I understand the v1 plugin has `# skip_database_creation = false` and you're trying to make it match. A negative setting makes a little more sense in v1 because db creation was the default from the start. I don't think we need to mimic the v1 setting so closely since the v2 plugin has been around for years and never defaulted to creating buckets. 

 Could we change this to `# create_buckets = false`?